packages/uhk-web/src/app/components/side-menu/side-menu.component.ts

```diff
@@ -67,6 +74,10 @@ export class SideMenuComponent implements OnInit, OnDestroy {
         }
     }
 
+    getMacroUsageCount(macroId) {
```

Comment:
it is not an optimal solution. If we have 10 macro that used in 20 key then it will 200 iteration over the 20 key action